SillyBusinessInc/SillyBusinessGame
# Origin/feat/#146 room teleportation based on room type

## Purpose of this PR:
 Teleporting between scenes should be based on the scene type aswell. in fact, the ID should not be a part of the scene at at. Instead it should load a random scene of type_X. we are not using Bonus room now, so the rooms except for bonus room will be loaded.

## How to test:
you can check the connected rooms and the scene change(ex. move from a scene to b scene) in the console, and for each room you can check in the door inspector(NextRoomType, NextRoomName etc).

### links: 
https://github.com/SillyBusinessInc/SillyBusinessGame/issues/146
